src/js/frontend/src/components/ChartTitle.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import Radium from "radium"
+import { Link } from 'react-router'
+
+let ChartTitle = React.createClass({
```

Comment:
Yeah - for some reason we had a weird mix of the two types of components even before I joined and messed everything up ;-)  But we seem to have more of the new style than the old, which is presumably good.